### ISSUE TITLE
UX issue when opening file from sidebar

### ISSUE THREAD

**Issue by snoopdouglas** [lapce/lapce#1398]
## Lapce Version

0.2.0

## System information

Fedora 36

## Describe the bug

When opening a file from the sidebar, the 'Open Editors' section expands immediately, pushing down the item you just clicked:

![image](https://user-images.githubusercontent.com/4892574/193006830-c5bfae4e-79b2-4319-8970-344f91f27fc4.png)

## Additional information

I don't find this often means that I'm accidentally opening the item above the one I originally clicked, but it doesn't feel right. So, I wonder whether we should either:

* Move 'Open Editors' to the bottom of the sidebar
* Only expand it once the mouse has left the sidebar (preferably with an animation so it's more obvious what just happened)
  * In this case, the 'Open Editors' header could 'blink' - and perhaps show a badge - to indicate that it's beginning to acquire items that've not yet been shown, ie.:

![image](https://user-images.githubusercontent.com/4892574/193008232-effd507c-9a32-4714-ad27-e4035a8dcdc8.png)

**Comment by panekj** [comment]
This is exactly same behaviour as in VS Code

**Comment by superlou** [comment]
As someone who habitually double-clicks to open a file, the shift always gets me to open two files. I guess I don't run into it in VSCode because I have open editors disabled:

![image](https://user-images.githubusercontent.com/709695/193026287-d6e8aed4-b727-4611-a4ae-0487afed3e69.png)

**Comment by snoopdouglas** [comment]
Yeah, I'm in the same boat as @superlou. IMO, we shouldn't be aiming for UX-issue-for-issue parity with VS Code. 😉

**Comment by NiceSieve** [comment]
> This is exactly same behaviour as in VS Code

I tried it in VSCode, and see a noticeable delay. Lapce is just too fast compared to VSCode 😉
There is also a second difference: VSCode has "half open" tabs (with italic title) for single clicked files, and "full open" tabs for double clicked or edited files. So the first half tab gives you a delay, and any further half tabs don't move the open editors section. The delay is what gives you that time double click.

**Comment by vintprox** [comment]
It is a bit of a hurdle here switching to new exercises from `rustlings` course. 😆 I want this "Open Editors" panel gone altogether, as a preference.

**Comment by JonahPlusPlus** [comment]
I made a fix for this by moving the "Open Editors" panel to be below the file tree and making it resizable.
I just have to make it so the default size is smaller and make it so that the split reacts when hovering.
I'll make a PR once I complete that and after #1574 is merged.
Once both PRs are merged, the "Open Editors" panel won't affect the file tree and would be able to be resized/collapsed.

**Comment by Bruce-Hopkins** [comment]
Is this issue still open?

**Comment by superlou** [comment]
Yes. Because you can collapse open editors, there is a workaround if you do not want to use open editors at all. However, you have to do this on each start of Lapce since Open Editors opens expanded (at least I couldn't find a setting for it).

**Comment by HaoliangXu** [comment]
Now the Open Editors panel has a fix height, guess we can close this issue?

**Comment by superlou** [comment]
It also looks like a setting has been added to remove the Open Editors panel:
![image](https://github.com/lapce/lapce/assets/709695/a1ced73f-1569-4ccb-8a30-5ca95aa3b81a)
